plasmapy/formulary/dimensionless.py

```diff
@@ -8,7 +8,14 @@
 highly turbulent, while turbulence is negligible at low Reynolds
 numbers.
 """
-__all__ = ["beta", "quantum_theta", "Reynolds_number", "Mag_Reynolds"]
+__all__ = [
+    "beta",
+    "quantum_theta",
+    "Reynolds_number",
+    "Re_",
+    "Mag_Reynolds",
+    "Rm_",
+]
```

Comment:
Can you please alphabetize all the all dunders?  The order of the dunders dictates the order of the documentation tables.

Reply:
Sure thing